integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonExtensionSpark2SqlParser.scala

```diff
@@ -56,4 +58,13 @@ class CarbonExtensionSpark2SqlParser extends CarbonSpark2SqlParser {
         CarbonSparkSqlParserUtil.loadDataNew(
           databaseNameOp, tableName, Option(optionsList), partitions, filePath, isOverwrite)
     }
+
+  /**
+   * REFRESH MATERIALIZED VIEW mv_name
+   */
+  private lazy val refreshTable: Parser[LogicalPlan] =
```

Comment:
why is this needed?

Reply:
Spark 3.1 has added Refresh table in SparkSqlParser and it fails with UnresolvedTableOrView. So, carbon need to handle it. @vikramahuja1001 check and remove from DDLStrategy if not used